webrtc-audio-processing-sys/build.rs

```diff
@@ -321,10 +335,13 @@ fn main() -> Result<()> {
         .includes(&include_dirs)
         .flag("-std=c++17")
         .flag("-Wno-unused-parameter")
-        .flag("-Wno-deprecated-declarations")
-        .flag("-Wno-nullability-completeness")
-        .out_dir(out_dir())
-        .compile("webrtc_audio_processing_wrapper");
+        .flag("-Wno-deprecated-declarations");
+
+    if cfg!(target_os = "macos") {
+        cc_build.flag("-Wno-nullability-completeness");
+    }
```

Comment:
DO we need this flag? Do we need it specifically on MacOS? 🤔

All the `-W...` flags just enable warnings, I guess these were instended as an aid when making changes to wrapper.h|cpp? Should be safe to drop these entirely if they cause problems.

Reply:
We might not, it's a macOS clang specific flag (or at least [recognized there](https://github.com/ddnet/ddnet/pull/8517#issuecomment-2185784633))

I'm not sure if it's inscope here but I'd be fine removing?

Reply:
Yeah I'd say fine to remove it.

But why have you made it specifical to macos in the first place? I see before it was applied unconditionally.

Reply:
It's unrelated, I just noticed that it was a warning we had never handled from this call. Because it was macos related I figured it would be good to handle here.